modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -42,7 +42,7 @@ If not set, the default value is based on `infrastructure.config.openshift.io/cl
 * Bare metal: `NodePortService`
 * Other: `HostNetwork`
 
-The `endpointPublishingStrategy` value cannot be updated.
+On GCP, you can configure the `endpointPublishingStrategy` value, `clientAccess`. Otherwise, the `endpointPublishingStrategy` value cannot be updated.
```

Comment:
I think we may want to be more specific here.
```suggestion
On GCP, you can configure the `endpointPublishingStrategy` subfield, `loadBalancer.providerParameters.gcp.clientAccess`. Otherwise, the `endpointPublishingStrategy` value cannot be updated.
```

Reply:
How does that sound to you?

Reply:
Good and yes clarifies some questions I had when writing.